data/quickstarts/oneapi-aikit-notebook.yaml

```diff
@@ -0,0 +1,95 @@
+        Use the **Intel oneAPI AI Analytics Toolkit** notebook with the **Default** container size.
+      review:
+        instructions: |-
+          #### Launched JupyterHub notebook server
+          Do you have a notebook browser tab?
+        failedTaskHelp: This task is not verified yet. Try the task again.
+      summary:
+        success: You have a notebook server
+        failed: Review the "Creating a Jupyter notebook" quick start for additional help.
+    - title: Review the Intel® oneAPI AI Analytics Toolkit Welcome Message
+      description: |-
+        ### Review the Intel® oneAPI AI Analytics Toolkit Welcome Message
+        1. Double-click the `AI_Kit_Welcome.ipynb` notebook that is available in the Intel® oneAPI AI Analytics Toolkit directory to open it in a new tab, if not already an open tab.
+        2. Review the contents of the AI Kit welcome message and its resources.
+      review:
+        instructions: |-
+          #### Reviewed the Intel® oneAPI AI Analytics Toolkit Welcome Message
+          Do you have the AI Kit welcome message open?
+        failedTaskHelp: This task is not verified yet. Try the task again.
+      summary:
+        success: You have opened the Intel® oneAPI AI Analytics Toolkit welcome message and read its contents. 
+        failed: Try the steps again.
+    - title: Load a Intel® oneAPI AI Analytics Toolkit sample notebook
+      description: |-
+        ### Loading a Intel® oneAPI AI Analytics Toolkit sample notebook
+        1. Click on the `oneAPI-samples.git` directory. As part of the Intel® oneAPI AI Analytics Toolkit on RedHat OpenShift Data Science, we have provided you three examples to help you get started with different AI Kit optimizations.
+        2. Click on the `E2E_use_case_with_Intel_optimized_XGBoost_daal4py` directory, then click on the `E2E_use_case_with_Intel_optimized_XGBoost_daal4py.ipynb` notebook.
+        3. Choose the `Intel SKLearn, XGBoost, & Modin`  JupyterHub Kernel Environment to launch the notebook in.
+      review:
+        instructions: |-
+          #### Verify that the notebook launched successfully
+          Do you have a notebook tab for the `E2E_use_case_with_Intel_optimized_XGBoost_daal4py.ipynb` notebook in your browser in the `Intel SKLearn, XGBoost, & Modin`  JupyterHub Kernel Environment?
+        failedTaskHelp:
+          This task is not verified yet. Try the task again.
+      summary:
+        success: You loaded the example notebook in the correct AI Kit JupyterHub Kernel environment.
+        failed: Try the steps again.
+    - title: Run a Intel® oneAPI AI Analytics Toolkit sample notebook
+      description: |-
+        ### Running a Intel® oneAPI AI Analytics Toolkit sample
+        1. Run the `E2E_use_case_with_Intel_optimized_XGBoost_daal4py.ipynb` notebook cell by cell, taking care not to skip any steps. Once you have completed the entire notebook, you will have learned about some AI Kit optimizations and fundamentals. 
```

Comment:
`E2E_use_case_with_Intel_optimized_XGBoost_daal4py.ipynb` is wrong. It should be `IntelPython_XGBoost_daal4pyPrediction.ipynb`

Reply:
I tried the notebook but it failed with these errors
~~~
---------------------------------------------------------------------------
OSError                                   Traceback (most recent call last)
/tmp/ipykernel_941/3744328891.py in <module>
----> 1 train_data, train_label, test_data, test_label, n_classes, n_features = load_higgs(990000, 10000)

/tmp/ipykernel_941/1994935491.py in load_higgs(nrows_train, nrows_test, dtype)
      4         url = "http://archive.ics.uci.edu/ml/machine-learning-databases/00280/HIGGS.csv.gz"
      5         myfile = requests.get(url)
----> 6         open('./data/batch/HIGGS.csv.gz', 'wb').write(myfile.content)
      7     print("Reading data set...")
      8     data = pd.read_csv("./data/batch/HIGGS.csv.gz", delimiter=",", header=None, compression="gzip", dtype=dtype, nrows=nrows_train+nrows_test)

OSError: [Errno 28] No space left on device
~~~
Looks it is because of small space. Could we choose a bigger size one?

Reply:
The space is something that need to be decided disk size allocated.
I set this to `Large` for now but I'm confident that `Medium` size should be good to run these notebooks.

I'll try them on my end too.

Reply:
Medium and Large set only cpu/memory not storage size.